src/Compilers/CSharp/Portable/Emitter/Model/PEAssemblyBuilder.cs

```diff
@@ -15,6 +18,7 @@ internal abstract class PEAssemblyBuilderBase : PEModuleBuilder, Cci.IAssemblyRe
     {
         private readonly SourceAssemblySymbol _sourceAssembly;
         private readonly ImmutableArray<NamedTypeSymbol> _additionalTypes;
+        private ImmutableDictionary<WellKnownMember, MethodSymbol> _embeddedAttributesConstructors;
```

Comment:
It is not going to be just two types. @Vsadov is already waiting for this PR to start using more attributes in other feature-work. And more will follow in the future for other C# features.

---
In reply to: [113572450](https://github.com/dotnet/roslyn/pull/18715#discussion_r113572450) [](ancestors = 113572450)